microsoft-edge/webview2/release-notes/index.md

```diff
@@ -88,8 +88,10 @@ For full API compatibility, this Release version of the WebView2 SDK requires We
 The following APIs have been promoted to Stable and are now included in this Release SDK.
 
 
-<!-- ------------------------------ -->
-* Added support for Web Notification, for non-persistent notifications.  The `NotificationReceived` event for `CoreWebView2` controls web notification handling, allowing customization or suppression by the host app.  Unhandled notifications default to WebView2's UI.
+<!-- ---------- -->
+###### Web Notification
```

Comment:
```suggestion
###### Web Notification API
```
alts:
* Web Notification
* Web Notification API
* Web Notification API for custom handling of notifications
* Web Notification API to customize or suppress handling of notifications

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3253&tabs=dotnetcsharp#web-notification

@victorhuangwq @peiche-jessica

Reply:
result (from top of PR main page - outline):
```
## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Save as -- NEW SECTION
#### Web notification handling -- NEW SECTION
#### Permissions
#### Context menus
#### Status bar
#### Fluent overlay scrollbars
#### User Agent
#### Autofill
#### Audio
#### ...
```

Reply:
result:
```
## 1.0.2739.15
#### Promotions
###### Web notification handling
###### Save as
```